src/scripts/dfnshelper.js

```diff
@@ -0,0 +1,104 @@
+import { ar, be, bg, ca, cs, da, de, el, enGB, enUS, es, faIR, fi, fr, frCA, he, hi, hr, hu, id, it, kk, ko, lt, ms, nb, nl, pl, ptBR, pt, ro, ru, sk, sl, sv, tr, uk, vi, zhCN, zhTW } from 'date-fns/locale';
+import globalize from 'globalize';
+
+    "use strict";
+
+    function getLocale()
+    {

```

Comment:
@MrTimscampi should it be like this?
```suggestion
    export function getLocale() {
```

Reply:
Same thing I said to @MrTimscampi applies here

Reply:
Same thing I said to @MrTimscampi applies here